proj1.js

```diff
@@ -1,39 +1,140 @@
+var qnbank = [{
+  sno: 1,
```

Comment:
Any particular reason you have `sno` property? Elements of an Array already have an index.